go.mod

```diff
@@ -6,19 +6,19 @@ toolchain go1.23.1
 
 require (
 	github.com/bufbuild/buf v1.30.0
-	github.com/crossplane/crossplane-runtime v1.16.0
+	github.com/crossplane/crossplane-runtime v1.17.0
 	github.com/go-json-experiment/json v0.0.0-20240815175050-ebd3a8989ca1
 	github.com/go-logr/logr v1.4.2
 	github.com/go-logr/zapr v1.3.0
 	github.com/google/go-cmp v0.6.0
 	github.com/pkg/errors v0.9.1
 	github.com/upbound/provider-aws v0.47.4
 	go.uber.org/zap v1.27.0
-	google.golang.org/grpc v1.62.0
+	google.golang.org/grpc v1.63.2
```

Comment:
https://github.com/crossplane/function-sdk-go/pull/170 had google.golang.org/grpc to v1.66.2 - is there any reason to not bump all the way to that version?

Reply:
Good catch, done.